### PR TITLE
added ref keyword and struct explanation

### PR DESCRIPTION
**Pull request by LoneXHuB**

### CODE REVIEW

File: c#.org [hunk 1]
```diff
@@ -1,929 +1,937 @@
...
+
+  docview/media/ezio/22508547508522A1/f/tuto/c#/intrestingg c# books/Great/Jeffrey Richter - CLR via C#, 4th Edition-Microsoft Press (2012).pdf
+  * VSCode shortcuts
+  ctrl + j hide the terminal pannel
+  ctrl + b hide and show the file explorer panel
+  ctrl + shift + p for executing commands
+
+  * Awesome.wm
+  just remove free desktop to make startup time faster
+* LoneX Toxic
```
Comment (revoltez) [inline c#.org:931]: not needed, remember this is objective truth

File: c#.org [hunk 2]
```diff
@@ -1,929 +1,937 @@
...
+    *C# basics
```
Comment (revoltez) [inline c#.org:932]: redundant , kayna lfoug, just add it like this
// "*" =h1 in html  " ** " =h2  a secondary heading and so on

File: c#.org [hunk 3]
```diff
@@ -1,929 +1,937 @@
...
+      by ref or by value :
```
Comment (revoltez) [inline c#.org:933]: kima hadi, should be added like this 
** by ref or by value

File: c#.org [hunk 4]
```diff
@@ -1,929 +1,937 @@
...
+        All values are passed by value unless specifically stated as a ref or out parameter. But you need to remember that a reference variable, i.e. a non-value-type variable such as an object, is already a reference. So even if you pass that reference variable into a method that takes a normal 'by value' argument (not a 'by ref' or 'out' argument), its actually a reference that you are passing in.
+      struct:
```
Comment (revoltez) [inline c#.org:935]: kif kif , should be a secondary heading under basics like this
** struct

**Review by revoltez — COMMENTED** [review]
it was an honour sir

**Comment by LoneXHuB** [comment]
> it was an honour sir

the pleasure was mine...

File: c#.org (hunk 1 above)
Comment (LoneXHuB) [reply c#.org:931]: it's just for organisation sake to find yourself fast

File: c#.org (hunk 2 above)
Comment (LoneXHuB) [reply c#.org:932]: this is some advanced coding stuff right there ... but I will try to follow it next time

File: c#.org (hunk 4 above)
Comment (LoneXHuB) [reply c#.org:935]: thank you for the knowledge

**Pull request closed without merging**